src/main/webapp/app/overview/course-card.component.ts

```diff
@@ -60,7 +60,14 @@ export class CourseCardComponent implements OnChanges {
     ngOnChanges() {
         if (this.course.exercises && this.course.exercises.length > 0) {
             this.exerciseCount = this.course.exercises.length;
-            const nextExercises = this.exerciseService.getNextExercisesForDays(this.course.exercises);
+            // sets the exercise with the earliest due date which is not yet successful (no 100%)
+
+            // debugger;
```

Comment:
I believe this should be removed.

Reply:
Thank you for noticing. I will be more careful next time. 👍

Reply:
When should I press the button "Resolve conversation" ? And how is it indicated that your requested changes have been addressed?  (Moreover, what does the button "Start Conversation/Discussion" do?^^)

Reply:
Usually just resolve the conversation when you have implemented the suggestion (or in case you rejected the suggestion). If you want to indicate that the requested changes are addressed go to the bottom of the page and expand the "X changes requested" tab. There you can dismiss the change request with a message (e.g. "code changed" or "changes implemented"):
![grafik](https://user-images.githubusercontent.com/72132281/117023676-9ca7aa00-acf9-11eb-8693-c82259f93723.png)

Reply:
Oh, Thank you very much for this detailed explanation!